molecule/config_upload_tdg/converge.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: 'Test file config'
+  hosts: cluster
+  become: true
+  become_user: root
+  gather_facts: false
+  vars:
+    cartridge_app_config_upload_mode: 'tdg'
+    cartridge_eval_body: >-
```

Comment:
Looks strange without `eval` step. Let's place these vars on each import separately.

Reply:
No. Eval is here.